conf/application.conf

```diff
@@ -183,7 +183,9 @@ feature {
   uniform-debug {
     enabled = true
   }
-
+  cr01 {
+    enabled = true
```

Comment:
Do we need to create config for prod to disable this feature?

Reply:
I believe so 👍

Reply:
https://github.com/hmrc/app-config-base/pull/4229